packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -190,13 +154,24 @@ const SubMenu = ({
     closeOnBlur,
     colorMode,
     mouseOnSubMenuTitle,
+    isCollapsable,
   };
 
   return (
     <SubMenuContext.Provider value={context}>
-      <PseudoBox as="li" role="none" display="flex" {...props}>
+      <PseudoBox
+        as="li"
+        role="none"
+        display="flex"
+        flexDirection={isCollapsable ? "column" : "row"}
+        {...props}
+      >
         {typeof children === "function"
-          ? children({ isOpen: _isOpen, onClose: closeMenu })
+          ? children({
+              isOpen: _isOpen,
+              onClose: closeMenu,
+              isCollapsable: isCollapsable,
+            })
```

Comment:
We dont want to allow, isCollapsable in renderProps. IMO this is unnecessary since we are already controlling it via prop